partition/roles/sonic/tasks/main.yaml

```diff
@@ -42,6 +42,11 @@
     src: resolv.conf.j2
     dest: /etc/resolv.conf
 
+- name: render docker daemon.json
```

Comment:
But then a handler must be triggered to restart the dockerd

Reply:
I am unsure if the changes in [the latest commit](https://github.com/metal-stack/metal-roles/pull/290/commits/4d10c3e055e811f2bf23c6a4c3c68a0c2b4dd6f6) are correct. Can you have another look?